libraries/lib-time-and-pitch/FormantShifterLogVisualization.py

```diff
@@ -0,0 +1,62 @@
+# For developers. To use this:
+#
+# 1. Create overrideLogTime.txt in lib-time-and-pitch's root directory,
+# 2. Enter a time in seconds, e.g. (0.5)
+# 3. Run your Audacity build and play a track with pitch shifting and formant preservation enabled.
+#
+# You will hear a gentle noise burst at the specified time, which indicates precisely the place where the snapshot was taken.
+# Now you can run this script from lib-time-and-pitch's root directory.
+
+import matplotlib.pyplot as plt
+import FormantShifterLog as FSL
```

Comment:
I think this should be moved into a separate directory, and with `requirements.txt`

Reply:
What is `requirements.txt` ?

Reply:
Your script has non-trivial dependencies. `requirements.txt` is a standard Python to define such dependencies.